v1/faasd.yaml

```diff
@@ -0,0 +1,50 @@
+description: A portable FaaS framework
+version: 0.11.4
```

Comment:
Right, that was one of the things I wanted your feedback on - any way we can make it pull down latest? So it doesn't need updating when there's a new release out?

Reply:
I am not sure how your code works, but anything is possible.

Given this is a kind of marketplace or package manager, it's best practice to offer versions.

Reply:
Since it's not meant to be a production deployment, does it make sense to provide multiple versions?

I'm really asking about a static `/latest/` URL or so that we could use rather than use a static version.